manifests/pulp/master.pp

```diff
@@ -0,0 +1,65 @@
+# The pulp master configuration
+#
+# This scenario assumes the pulp master is already configured otherwise,
+# usually via puppet-katello. Here we only configure the qpid router (if
+# desired) and crane.
+class foreman_proxy_content::pulp::master (
+  Boolean $qpid_router = $::foreman_proxy_content::qpid_router,
+  String $hub_host = $::foreman_proxy_content::qpid_router_hub_addr,
+  Integer[0, 65535] $hub_port = $::foreman_proxy_content::qpid_router_hub_port,
+  String $broker_host = $::foreman_proxy_content::qpid_router_broker_addr,
+  Integer[0, 65535] $broker_port = $::foreman_proxy_content::qpid_router_broker_port,
+) {
+  if $qpid_router {
```

Comment:
Maybe this should be it's own manifest? Pulp does not actually talk to the router and you may choose to deploy the router separately from your Pulp host or even your qpid host. That is, you could have Pulp server, Qpid server and Qdrouter running all separately or router and qpid paired.

Reply:
I think you're right.